src/BaGet.Core/ServiceIndex/BaGetServiceIndex.cs

```diff
@@ -6,10 +6,15 @@
 
 namespace BaGet.Core
 {
-    public class BaGetServiceIndex : IServiceIndexService
+    /// <summary>
+    /// The NuGet Service Index service, used to discover other resources.
+    /// 
+    /// See https://docs.microsoft.com/en-us/nuget/api/service-index
+    /// </summary>
+    public class BaGetServiceIndex
```

Comment:
I think we can rename this to just `ServiceIndex` now? I believe I called this type `BaGetServiceIndex` because the `ServiceIndexClient` type used to be called `ServiceIndex`

Reply:
sure